### DISCUSSION TITLE
OpenSPH visualization

### DISCUSSION THREAD

**Discussion by pavelsevecek** [christophmschaefer/miluphcuda#5] [Show and tell]
Hello there,
I had a brief look at your code. Impressive work! I'll definitely study it more.
For easier comparison with my results, I added a reader of your .h5 files to my code/visualizer.
I believe it's quite convenient, hopefully, it could be useful for others, too. 
Here is a short preview of what it can do:
https://www.youtube.com/watch?v=-_XzELkA5h4

If you are interested, have a look at https://gitlab.com/sevecekp/sph
Any feedback is highly appreciated.

**Comment by christophmschaefer** [comment]
Hi Pavel,

wow, that's awesome! Thanks, I'll have a closer look!
